dev/deps/spark-deps-hadoop-2.7-hive-2.3

```diff
@@ -27,10 +24,10 @@ avro-ipc/1.10.2//avro-ipc-1.10.2.jar
 avro-mapred/1.10.2//avro-mapred-1.10.2.jar
 avro/1.10.2//avro-1.10.2.jar
 blas/2.2.0//blas-2.2.0.jar
-bonecp/0.8.0.RELEASE//bonecp-0.8.0.RELEASE.jar
 breeze-macros_2.12/1.2//breeze-macros_2.12-1.2.jar
 breeze_2.12/1.2//breeze_2.12-1.2.jar
 cats-kernel_2.12/2.1.1//cats-kernel_2.12-2.1.1.jar
+checker-qual/3.8.0//checker-qual-3.8.0.jar
```

Comment:
In general, for all new binaries, we need to check the binary license and update `LICENSE-binary`.
If we don't use this, can we cut this transitive dependencies?

Reply:
sure, let me update these.